ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -478,8 +477,10 @@ private class JdbcLedgerDao(
   )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
     dbDispatcher.executeSql(metrics.daml.index.db.storeRejectionDbMetrics) {
       queries.withAsyncCommit { implicit conn =>
-        for (info @ SubmitterInfo(_, _, commandId, _) <- submitterInfo) {
-          stopDeduplicatingCommandSync(domain.CommandId(commandId), info.singleSubmitterOrThrow())
+        for (info <- submitterInfo) {
+          stopDeduplicatingCommandSync(
+            domain.CommandId(info.commandId),
+            info.singleSubmitterOrThrow())
           prepareRejectionInsert(info, offset, recordTime, reason).execute()
```

Comment:
This is more for my understanding: why don't we store a rejection if a `submitterInfo` is missing? Do we do something else instead?

Reply:
Very good question! If `SubmitterInfo` is missing, it means that this transaction originated on a remote participant. Hence, this participant cannot keep track of the command, as this is done exclusively on the local participant. Does this make sense to you?

Reply:
It does, thanks 👍